lib/msf/core/exploit/http/client.rb

```diff
@@ -368,29 +396,53 @@ def send_request_cgi(opts={}, timeout = 20, disconnect = true)
       actual_timeout =  opts[:timeout] || timeout
     end
 
-    print_line("*" * 20) if datastore['HttpTrace']
```

Comment:
AFAIK, this was just a debugging print. I've removed it, as it wasn't even in `send_request_raw`.